[2022-12-04T16:14:34.323+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: data_generation.notification scheduled__2022-12-04T10:00:00+00:00 [queued]>
[2022-12-04T16:14:34.332+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: data_generation.notification scheduled__2022-12-04T10:00:00+00:00 [queued]>
[2022-12-04T16:14:34.333+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2022-12-04T16:14:34.349+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2022-12-04T16:14:34.350+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2022-12-04T16:14:34.362+0000] {taskinstance.py:1304} INFO - Executing <Task(BashOperator): notification> on 2022-12-04 10:00:00+00:00
[2022-12-04T16:14:34.369+0000] {standard_task_runner.py:55} INFO - Started process 570 to run task
[2022-12-04T16:14:34.373+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'data_generation', 'notification', 'scheduled__2022-12-04T10:00:00+00:00', '--job-id', '72', '--raw', '--subdir', 'DAGS_FOLDER/generate_data.py', '--cfg-path', '/tmp/tmphwiv9dkk']
[2022-12-04T16:14:34.376+0000] {standard_task_runner.py:83} INFO - Job 72: Subtask notification
[2022-12-04T16:14:34.444+0000] {task_command.py:389} INFO - Running <TaskInstance: data_generation.notification scheduled__2022-12-04T10:00:00+00:00 [running]> on host c7dedbf79ea9
[2022-12-04T16:14:34.502+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_generation
AIRFLOW_CTX_TASK_ID=notification
AIRFLOW_CTX_EXECUTION_DATE=2022-12-04T10:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-12-04T10:00:00+00:00
[2022-12-04T16:14:34.504+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2022-12-04T16:14:34.509+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo received data at 2022-12-04']
[2022-12-04T16:14:34.522+0000] {subprocess.py:86} INFO - Output:
[2022-12-04T16:14:34.525+0000] {subprocess.py:93} INFO - received data at 2022-12-04
[2022-12-04T16:14:34.526+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2022-12-04T16:14:34.551+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=data_generation, task_id=notification, execution_date=20221204T100000, start_date=20221204T161434, end_date=20221204T161434
[2022-12-04T16:14:34.585+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2022-12-04T16:14:34.603+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
